doc: removing risc0 references for SP1 (#365)
InfinityVM/InfinityVM: programs/README.md

This folder contains sample programs for running in the zkVM executor.

> We currently only have an SP1 program, but might support other zkVMs in the future. 

Reviewer: ```suggestion
We currently only have an SP1 program, and plan to support other zkVMs in the future. 
```
